src/Metrics.tcl

```diff
@@ -254,5 +254,250 @@ proc report_design_area_metrics {args} {
   utl::metric_float "design__instance__utilization__stdcell" $stdcell_util
 }
 
+proc report_puts { out } {
+  upvar 1 when when
+  upvar 1 filename filename
+  set fileId [open $filename a]
+  puts $fileId $out
+  close $fileId
+}
+
+define_cmd_args "report_metrics" {[-stage][-when][-include_erc][-include_clock_skew][-metrics_report_dir]}
```

Comment:
[-stage][-when] are both mandatory args not switches

[-metrics_report_dir] should be [-metrics_report_dir dir]